skafka/src/main/scala/com/evolutiongaming/skafka/producer/ProducerConfig.scala

```diff
@@ -46,12 +46,16 @@ final case class ProducerConfig(
       (C.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION, maxInFlightRequestsPerConnection.toString),
       (C.INTERCEPTOR_CLASSES_CONFIG, interceptorClasses mkString ","),
       (C.ENABLE_IDEMPOTENCE_CONFIG, idempotence.toString),
-      (C.TRANSACTION_TIMEOUT_CONFIG, transactionTimeout.toMillis.toString)
     )
 
-    val bindings = bindings1 ++
-      transactionalId.map { (C.TRANSACTIONAL_ID_CONFIG, _) }
-    partitionerClass.map { value => (C.PARTITIONER_CLASS_CONFIG, value) }
+    val transactions = transactionalId.toList.flatMap { id =>
```

Comment:
just curious, does this work without `.toList` ?

Reply:
No, `Option.flatMap` expects only `Option`